CHANGELOG.md

```diff
@@ -33,6 +33,8 @@ Bug Fixes
   where the resulting object did not set the index attribute properly. (#694)
 - Fixed an error in the `CCDNoise.getVariance()` function, as well as some
   errors in the documentation about the units of CCDNoise parameters. (#713)
+- Fixed a bug where InterpolatedKImage incorrectly checked that its input real
```

Comment:
The grammar of this sounds like it used to check that the image was Hermitian, but that that check was incorrect, so now it doesn't.  That's now what you meant though.  (It was an and/or mixup.)  Maybe
"Fixed a bug in how InterpolatedKiImage checked for properly Hermitian input images."